### PR TITLE
do not re-run blocks from the leadership service

### PR DESCRIPTION
**Pull request by eugene-babichenko**
depends on input-output-hk/chain-libs#543

### CODE REVIEW

File: jormungandr/src/main.rs [hunk 1]
```diff
@@ -217,6 +218,7 @@ fn start_services(bootstrapped_node: BootstrappedNode) -> Result<(), start_up::E
                 fragment_msgbox,
                 enclave,
                 block_msgbox,
+                rewards_report_all,
```
Comment (mzabaluev) [inline jormungandr/src/main.rs:221]: Looks like this module begins to want a config struct or something.

**Review by mzabaluev — APPROVED** [review]

File: jormungandr/src/main.rs (hunk 1 above)
Comment (eugene-babichenko) [reply jormungandr/src/main.rs:221]: Well, it does. In one of later patches it starts to trigger a clippy warning.

**Comment by eugene-babichenko** [comment]
Since we have merged https://github.com/input-output-hk/chain-libs/pull/536 a bit ahead of time, this needs to wait until https://github.com/input-output-hk/jormungandr/pull/3133 which is compatible with the `rand` update is merged. @Zeegomo any chance to merge it soon?

**Comment by eugene-babichenko** [comment]
Nevermind, we'll create a support branch for poldercast 0.13.x

**Pull request merged**